src/bootstrap/datatable.html

```diff
@@ -52,7 +52,7 @@
 
         <!-- Columns -->
         <td repeat.for="columnLabel of columnLabels">
-          <a click.delegate="selected(row)">${displayValue(row, columnLabel.column) | convertManager: columnLabel.converter}</a>
+          <a click.delegate="selected(row)" innerhtml.bind="displayValue(row, columnLabel.column) | convertManager: columnLabel.converter"></a>
```

Comment:
i see. a separate commit (message) would have been nice though. 

i wounder if not `| sanitize` could/should be added. thoughts?